fly/commands/workers.go

```diff
@@ -191,3 +194,29 @@ func (w *worker) VersionCell() ui.TableCell {
 
 	return column
 }
+
+func (w *worker) ageCell() ui.TableCell {
+	var column ui.TableCell
+
+	const minute = 60
+	const hour = minute * 60
+	const day = hour * 24
+
+	age := time.Now().Unix() - w.StartTime
+	if w.StartTime <= 0 || age < 0 {
+		column.Contents = "na"
```

Comment:
for consistency with other "missing values" in fly -- maybe we should have a global constant for this?
```suggestion
		column.Contents = "n/a"
```

Reply:
Good suggestion, I'll update to `n/a`.